README.md

```diff
@@ -115,40 +115,63 @@ Most `google-cloud` libraries require a project ID.  There are multiple ways to
 Authentication
 --------------
 
-First, ensure that the necessary Google Cloud APIs are enabled for your project. To do this, follow the instructions on the [authentication document](https://github.com/GoogleCloudPlatform/gcloud-common/blob/master/authentication/readme.md#authentication) shared by all the gcloud language libraries.
+`google-cloud-java` uses
+[https://github.com/google/google-auth-library-java](https://github.com/google/google-auth-library-java)
+to authenticate requests. `google-auth-library-java` supports a wide range of authentication types,
+see the project's [README](https://github.com/google/google-auth-library-java/blob/master/README.md)
+and [javadoc](http://google.github.io/google-auth-library-java/releases/0.6.0/apidocs/) for more
+details.
+
+To access Google Cloud services, you first need to ensure that the necessary Google Cloud APIs are
+enabled for your project. To do this, follow the instructions on the
+[authentication document](https://github.com/GoogleCloudPlatform/gcloud-common/blob/master/authentication/readme.md#authentication)
```

Comment:
https://cloud.google.com/docs/authentication contains code for Java showing how to authenticate with the Java apiary. Referencing it for here might just be confusing.